docker/cli
# Update issues form

**- What I did**
Update the repository issues form to support bug and features requests.

Additionally, added links to other resources to guide users across support documentation and forums.

Signed-off-by: Bruno Sousa <bruno.sousa@docker.com>

**- How I did it**
Using the latest features available on GitHub to manage [issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) and using the same template from the [Moby Project](https://github.com/moby/moby/).

**- How to verify it**
Create a new issue after the PR is merged and confirm you are given the options to create either a bug or a feature request.

**- Description for the changelog**
Update the repository issues form to support bug and features requests.


**- A picture of a cute animal (not mandatory but encouraged)**
![9eec8720-13b3-41fd-8068-74f6fcbd3e4f](https://user-images.githubusercontent.com/107440821/185171150-39827ce4-e284-41ae-8ab2-6a76a86dddc0.JPG)
Gaspar :heart:
